vector/src/Data/Vector/Generic.hs

```diff
@@ -2473,7 +2471,10 @@ unstreamR s = new (New.unstreamR s)
   streamR (new (New.transformR f g m)) = inplace f g (streamR (new m))  #-}
 
 
-
+-- | Load a monadic stream bundle into a newly allocated vector. This function goes through
+-- a list, so prefer using `unstream`, unless you need to be in a monad.
+--
+-- @since 0.12.2.0
```

Comment:
[This doc comment is present in the version on Hackage](https://hackage.haskell.org/package/vector-0.12.3.0/docs/Data-Vector-Generic.html#v:unstreamM), but I didn't find it in the git history. Not sure what's going on here, but I added it here.

Reply:
That was an oversight. Thank you for catching it.